src/Notepads/Notepads.csproj

```diff
@@ -456,6 +456,16 @@
       <AppxManifest Include="$(GeneratedAppxManifest)" />
     </ItemGroup>
   </Target>
+  <Target Name="AssignAppCenterSecret" BeforeTargets="BeforeBuild" Condition=" $(AppCenterSecret) != '' ">
+    <PropertyGroup>
+      <InputFile>App.xaml.cs</InputFile>
+    </PropertyGroup>
+    <WriteLinesToFile
+        File="$(InputFile)"
+        Lines="$([System.IO.File]::ReadAllText($(InputFile)).Replace('AppCenterSecret = null','AppCenterSecret = &quot;$(AppCenterSecret)&quot;'))"
+        Overwrite="true"
+        Encoding="Unicode"/>
+  </Target>
```

Comment:
@JasonStein This target will handle replacing `AppCenterSecret` with actual value. The actual value is passed on to MSBuild as an environment variable with the same name from Github secrets.

Reply:
Awesome! I will add a comment in the code as well.